clusterdock/models.py

```diff
@@ -117,8 +117,13 @@ def start(self, network, pull_images=False, update_etc_hosts=True):
                 raise DuplicateHostnamesError(duplicates=duplicate_hostnames,
                                               network=self.network)
 
+        if clusterdock_args and clusterdock_args.clusterdock_config_directory:
```

Comment:
i would think this `clusterdock_config_host_dir` creation needs to move to `__init__`

Reply:
Removed it as moved the logic in Node.start to Node.__init__ where the args are available.